build/teamcity-bazel.sh

```diff
@@ -10,6 +10,12 @@ tc_prepare
 export TMPDIR=$root/artifacts
 mkdir -p "$TMPDIR"
 
+# Sets `-test.v` in Go tests. (This is a no-op for now since we're not
+# running tests with Bazel in CI, but we shortly will, and we'll want to
+# make sure this is included when we do.)
+# Ref: https://github.com/bazelbuild/rules_go/pull/2456
+echo 'test --test_env=GO_TEST_WRAP_TESTV=1' > $root/.bazelrc.user
```

Comment:
do you think making some sort of `.bazelrc.ci.user` file, and do `cp .bazelrc.ci.user .bazelrc.user` (or something similar such that bazel uses this profile in CI) would make managing a CI user file more clear?

Reply:
Smart, done. (Went with `.bazelrc.ci` instead of `.bazelrc.ci.user`, seems to make a little more sense?)